release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2913,3 +2913,36 @@ $ oc adm release info 4.11.9 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-12"]
+=== RHSA-2022:7201 - {product-title} 4.11.12 bug fix and security update
+
+Issued: 2022-10-31
+
+{product-title} release 4.11.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:7201[RHSA-2022:7201] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7200[RHBA-2022:7200] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.12 --pullspecs
+----
+
+[id="ocp-4-11-12-known-issues"]
+==== Known issues
+
+* When upgrading PTP Operator from 4.10 to 4.11, OpenShift-ptp subscription reports the error: `no channel heads found in channel stable of package ptp-operator`. However, the operator correctly updates. (link:https://issues.redhat.com/browse/OCPBUGS-2042[*OCPBUGS-2042*])
+
+* If a cluster is incrementally updated from a version less than or equal to 4.9, the `namespace` `openshift-dns` may not contain the required pod-security labels required for future version updates. (link:https://issues.redhat.com/browse/OCPBUGS-1549[*OCPBUGS-1549*])
```

Comment:
I think 
```suggestion
* If a cluster is incrementally updated from a version less than or equal to 4.9, the `openshift-dns` `namespace` may not contain the required pod-security labels required for future version updates. (link:https://issues.redhat.com/browse/OCPBUGS-1549[*OCPBUGS-1549*])
```
reads a bit more clearer since `openshift-dns` is the name of the `namespace`. Ignore if thats not the case, that was just how I read it!